staff/alex-martinez/precourse/airlines/airlinesPRO/index.js

```diff
@@ -0,0 +1,195 @@
+function Vuelo(id,to,from,cost,scale){
+    this.id = id;
+    this.to = to;
+    this.from = from;
+    this.cost = cost;
+    this.scale = scale;
+}
+
+//función para poner la primera letra del usuario en mayúscula
+function PrimeraMayus(string){
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+//funcion inicial inicio sesión
+function saludo(){
+    //pasamos el nombre de usuario a minuscula + eliminar espacios
+    var usuario = prompt('Escibe tu usuario').toLowerCase().trim();
+    
+
+    if(usuario){
+        console.log(`Bienvenido ${PrimeraMayus(usuario)}`);
+    }
+}
+//función para mostrar los vuelos
+function vuelos(){
+    console.log('Vuelos de hoy: ')
+    for(var i = 0; i<flights.length; i++){
+        if(flights[i].scale === false){
+            var escala = 'no realiza ninguna escala';
+        }else{
+            escala = 'si realiza escala';
+        }
+
+        console.log(`El vuelo con origen ${flights[i].from} y destino ${flights[i].to} tiene un coste de ${flights[i].cost}€, y ${escala}`);
+    }
+}
+
+//función coste medio
+function costeMedioVuelos(){
+    //variable que sumará los precios de los vuelos
+    var costeTotal = 0;
+    //variable que almacenará el coste medio
+    var costeMedio = 0;
+    for(var i = 0; i<flights.length; i++){
+        //variable para acumular todos los costes
+        var costes = flights[i].cost;
+        //suma de los precios
+        costeTotal += costes;
+    }
+    //coste total / numero de total de vuelos, con dos decimales
+    costeMedio = parseFloat(costeTotal/flights.length).toFixed(2);
+    console.log(`El coste medio de los vuelos es de ${costeMedio}€`);
+}
+
+function vuelosEscalas(){
+    //array para guardar los vuelos con escala y despues contarlos
+    var conEscala = [];
+    console.log('Vuelos con escala: ')
+    for(var i=0; i<flights.length; i++){
+        if(flights[i].scale === true){
+            conEscala.push(flights[i]);
+            console.log(`Vuelo con origen ${flights[i].from} y destino ${flights[i].to}`);   
+        }    
+    }
+    console.log(`En total hay ${conEscala.length} vuelos con escala`);
+}
+
+//función para mostrar los 5 ultimos vuelos
+function ultimosCinco(){
+    //variable que almacena los 5 ultimos vuelos del array
+    var ultimosVuelos = flights.slice(flights.length-5);
+    console.log('Los últimos 5 vuelos del día son: ')
+    for(var i =0; i<ultimosVuelos.length; i++){
+        //console.log(ultimosVuelos[i]);
+        console.log(`El vuelo con origen ${ultimosVuelos[i].from} y con destino ${ultimosVuelos[i].to}`);
+    }
+}
+
+//función principal
+function airline(){
+    saludo();
+    console.log('');
+    vuelos();
+    console.log('')
+    costeMedioVuelos();
+    console.log('')
+    vuelosEscalas();
+    console.log('')
+    ultimosCinco();
+    console.log('PRO');
+    preguntaAdminUser(); 
+     
+}
+
+airline();
+
+//PRO//
+
+//contar vuelos para sumar id automatico
+function countVuelos(){
```

Comment:
I suggest the whole name in english

Reply:
yes I agree with you. 
It is best practice to use English words in the code. Thanks mate.